### ISSUE TITLE
Create issues for "To do" from extension readmes

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/ocds-extensions#85]
I've removed the following from extension readmes, as todo's are easily forgotten in readme files, and ought to be created as issues instead. Since these todo's are quite old and obscure, I don't know if they are still relevant or what they mean.

# Budget breakdown

* ~~Finalize guidance on use of extended fields in the core `planning.budget` field.~~
* ~~Finalize guidance where source entity is not known at time of budgeting~~

See issue [#377](https://github.com/open-contracting/standard/issues/377).

# Exchange rate

* Add pattern properties to validate `amount_CODE` fields. #105

# Metrics

* ~~Check oneOf schema syntax for value **or** quantity~~
* Add relatedRequirementID to requirements extension #107
* ~~Validate schema updates~~

# Performance failures

* Add a `relatedTransactions` reference to the performance failure object? #108

# Requirements

The extension does not describe formulae for calculated computed values.

The extension does not describe whether data should be published openly or not.

**Comment by duncandewhurst** [comment]
**budget breakdown** - I'm not sure what additional guidance was envisaged, since this seems to be covered by the [guidance](https://github.com/open-contracting-extensions/ocds_budget_breakdown_extension#guidance) section of `README.md`, so I don't think we need an issue for those.

**exchange rate** - I've opened https://github.com/open-contracting/ocds-extensions/issues/105

**Comment by duncandewhurst** [comment]
> # Metrics
> 
>     * Check oneOf schema syntax for value **or** quantity

I'm guessing here but I think Tim was considering whether to use `oneOf` to allow either a value object or a numeric value to be specified as a measure. I don't think we need an issue for this as a different approach was taken (separate properties for `measure` (non-financial) and `value` (financial)).

Whilst I looking at this I spotted an error in the example in the readme, which still uses `quantity` so I've opened https://github.com/open-contracting/ocds-extensions/issues/106

>     * Add relatedRequirementID to requirements extension

See https://github.com/open-contracting/ocds-extensions/issues/107

>     * Validate schema updates

I think this is now covered by the tests on extension schemas.

> # Performance failures
> 
>     * Add a `relatedTransactions` reference to the performance failure object?

See https://github.com/open-contracting/ocds-extensions/issues/108

> # Requirements
> 
> The extension does not describe formulae for calculated computed values.
> 
> The extension does not describe whether data should be published openly or not.

These points were included in the readme to make explicit which of the requirements expressed in the [original Github issue](https://github.com/open-contracting/standard/issues/223#issue-98373063) weren't addressed by the extension. I'm not sure they need new issues opening, but I do think it's relevant to state that somewhere.
